code/datums/mind.dm

```diff
@@ -187,6 +186,8 @@
 /datum/mind/proc/gen_objective_text(admin = FALSE)
```

Comment:
You're gonna have to listify this proc too, and any procs that call it will need tweaking using `Join()`.